### PR TITLE
Add edge support

### PR DESCRIPTION
**Pull request by birdsarah**
Fixes: #103

Have tested this on a windows machine and works fine.

### CODE REVIEW

**Comment by birdsarah** [comment]
Not sure what these test failures are - would appreciate some guidance.

**Comment by davehunt** [comment]
@birdsarah these failures look to be related to the Selenium python client issue that you've addressed: `AttributeError: 'str' object has no attribute 'decode'`. I think we can ignore them for now, and hopefully get a new client release out soon.

**Comment by davehunt** [comment]
Could you add a test for Edge? We won't be able to run it on Travis, but might ultimately be able to get it to run on AppVeyor. We'll at least be able to run it locally when on a Windows machine with Edge and EdgeDriver. See the Safari test for an example: https://github.com/pytest-dev/pytest-selenium/commit/03cd43e5e647b9e1c03e1d7da191cb6964537e14

**Comment by birdsarah** [comment]
(just so you know I'll be away for 10 days - will get back to this early april)

File: docs/user_guide.rst [hunk 1]
```diff
@@ -142,6 +142,23 @@ option to indicate where it can be found:
...
+
+To use Edge, you will need to
+`download Edge WebDriver <https://docs.microsoft.com/en-us/microsoft-edge/dev-guide/tools/webdriver>`_
+and specify ``Edge`` for the ``--driver`` command line option. If the driver
+executable is not available on your path, you can use the ``--driver-path``
+option to indicate where it can be found:
+
+.. code-block:: bash
+
+  $ py.test --driver Edge --driver-path /path/to/edgedriver
```
Comment (davehunt) [inline docs/user_guide.rst]: I've recently removed the `bash` from the command line example in the documentation. Also, the path should probably be `\path\to\MicrosoftWebDriver.exe` to represent the actual executable name and a Windows path.

File: docs/user_guide.rst [hunk 2]
```diff
@@ -142,6 +142,23 @@ option to indicate where it can be found:
...
+
+Selenium v2.47 or greater is required.
```
Comment (davehunt) [inline docs/user_guide.rst]: pytest-selenium depends on Selenium 3.x already, so we can remove this note.

**Comment by birdsarah** [comment]
Hi, sorry for delay. This week's looking busy.....trying to be realistic....feel free to jump in.

**Comment by davehunt** [comment]
Closing this in favour of #121, which addresses my comments. It took me a while to get a VM up and running but I was able to test this successfully. Unfortunately there's no way to test Edge in CI at the moment, so the test will have to be run manually if there's a risk of regression.
